entity/courier.py

```diff
@@ -0,0 +1,32 @@
+from entity.abstract_storage import AbstractStorage
+from entity.request import Request
+
+
+class Courier:
+    def __int__(self, request: Request, storages: dict[str, AbstractStorage]):
+        self.__request = request
+
+        if self.__request.departure in storages:
+            self.__from = storages[self.__request.departure]
+
+        if self.__request.destination in storages:
+            self.__to = storages[self.__request.destination]
+
+    def move(self):
+        self.__from.remove(name=self.__request.product, amount=self.__request.amount)
+        print(f"""
```

Comment:
Странное форматирование)
```suggestion
        print(
            f'Курьер забрал {self.__request.amount}'
            f'{self.__request.product}'
            f'из {self.__request.departure}'
        )
```